Integrations/integration-UrlScan.yml

```diff
@@ -6,6 +6,7 @@ display: urlscan.io
 description: Urlscan.io reputation
+defaultEnabled: false
```

Comment:
Kind of weird that we are disabling this and it didn't have "defaultEnabled:true" to start with. Please check with Server what logic was enabling this and if this is ok.

Reply:
@glicht 
there was no server logic that was enabling it, it's not enabled by default, so basically this change is redundant